From 08c154d48af952e5ab674a1a8cedbd50d7fb70cf Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 1 Sep 2019 21:34:14 -0500
Subject: gc.test: after-gc-hook - mark unresolved on failure (everywhere)

Previously this was done for just mips and mipsel,
59d9bcd468aab0d97d763595fd4e934044dc7590 "gc.test: after-gc-hook - mark
unresolved on failure for mips(el)", and then
fd4ba18bca1c6000fc0dd417a5b489e1ac60e0d9 "Fix gc.test "after-gc-hook
gets called" failures" attempted to fix it upstream, but as of 2.2.6
it's failing again, this time on amd64, so just mark it as unresolved
everywhere for now.

Bug: https://debbugs.gnu.org/31776
Bug-Debian: https://bugs.debian.org/900652
---
 test-suite/tests/gc.test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test-suite/tests/gc.test b/test-suite/tests/gc.test
index 1827599a4..6a790d108 100644
--- a/test-suite/tests/gc.test
+++ b/test-suite/tests/gc.test
@@ -67,7 +67,7 @@
       (add-hook! after-gc-hook thunk)
       (gc)
       (remove-hook! after-gc-hook thunk)
-      foo))
+      (maybe-gc-flakiness foo)))
 
   (pass-if "Unused modules are removed"
     (let* ((guard (make-guardian))
